packages/runtime/src/components/core/Stack.tsx

```diff
@@ -65,7 +86,18 @@ const Stack = React.forwardRef<HTMLDivElement, StackProps & { children: ReactNod
           ${cssStyle}
         `}
       >
-        {children}
+        {childs.map((child, idx) => {
+          return (
+            <div
```

Comment:
Wrappering with div will break the structure and make children's some css not working, like `flex: 1 1 auto`... 
Can we use css selector to add style to children? like `:last-child `